RecoMuon/MuonIdentification/plugins/MuonIdProducer.cc

```diff
@@ -445,7 +445,7 @@ void MuonIdProducer::produce(edm::Event& iEvent, const edm::EventSetup& iSetup)
 
   // muons first - no cleaning, take as is.
   if (muonCollectionHandle_.isValid()) {
-    for (const auto muon : *muonCollectionHandle_) {
+    for (const auto& muon : *muonCollectionHandle_) {
```

Comment:
this is a duplicate of https://github.com/cms-sw/cmssw/pull/29568/files#diff-89dc8f122700a0c3b857022fe730fd42R448
please revert or merge the two PRs

Reply:
#29568 is now merged in this PR.